flutter_map_compass/README.md

```diff
@@ -28,6 +28,8 @@ dependencies:
 ## Usage
 
 ```dart
+import 'package:flutter_map_compass/flutter_map_compass.dart';
+
```

Comment:
As now, the import statement would be inside of the class. Could you add it in a separate code block please?